server/services/config/crm4DisplayConfig.json

```
@@ -507,6 +507,18 @@
         }
       ]
     },
+    {
```

As Further Information is similar to the Evidence section, in that it does not always have data. We can use the same technique to hide the section when the data is empty.
Suggested change:-
```suggestion
    {
      "sectionId": "further-information",
      "title": "Further Information",
      "hideWhen": [
        {
          "apiField": "FurtherInformation",
          "equals": ""
        }
      ],
      "subsections": [
        {
          "customDisplay": {
            "name": "crmFurtherInformation",
            "apiField": "FurtherInformation"
          }
        }
      ]
    },
```